proto/eraftpb.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+package eraftpb;
+
+option java_package = "com.pingcap.tikv.eraftpb";
```

Comment:
Do we still need it?

Reply:
Yes. Otherwise I don't think TiSpark can use kvproto anymore.

Reply:
I haven't found any reference to eraftpb in TiSpack.
https://github.com/pingcap/tispark/search?utf8=%E2%9C%93&q=eraftpb&type=

Reply:
It's an indirect dependency. It depends on pdpb, which depends on eraftpb.

Reply:
Any suggestions? @ilovesoup

Reply:
ping @ilovesoup

Reply:
Just did a test seems fine for TiSpark. Please feel free go on.

Reply:
@BusyJay 

We can remove this now.